azure-pipelines.yml

```diff
@@ -485,7 +485,7 @@ jobs:
   displayName: Build Homebrew Formula
 
   dependsOn: BuildPythonWheel
-  condition: and(succeeded(), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'Manual'))
```

Comment:
No need to build at those trigger condition?

Reply:
yes, will revert these changes before merge it. Just test whether these pipelines can pass.